lib/Sapi.php

```diff
@@ -89,8 +89,6 @@ public static function sendResponse(ResponseInterface $response)
         if (null !== $contentLength) {
             $output = fopen('php://output', 'wb');
             if (is_resource($body) && 'stream' == get_resource_type($body)) {
-                if (PHP_INT_SIZE > 4) {
-                    // use the dedicated function on 64 Bit systems
                     // a workaround to make PHP more possible to use mmap based copy, see https://github.com/sabre-io/http/pull/119
```

Comment:
The indent of this block now needs to be reduced. code-style checks are failing.